app/support/satisfaction.js

```diff
@@ -1,11 +1,19 @@
-window.onload = function () {
+$(function() {
   var satisfaction = getQueryVariable('satisfaction');
   var origin = getQueryVariable('origin');
   document.getElementById('result').value = satisfaction;
   document.getElementById('origin').value = origin;
   document.getElementById('ip').value = myip;
-  document.getElementById('satisfaction').submit();
-}
+  var formData = $('form').serialize();
```

Comment:
Could be argued that this `formData` variable is not necessary considering you're only using it once. You could just plug `$('form').serialize()` into the `ajax` call.